src/index.ts

```diff
@@ -1,3 +1,24 @@
+interface AsyncStorage {
+    getItem: (key: string) => Promise<string | undefined | null>;
+    setItem: (key: string, value: string) => Promise<unknown>;
+    removeItem: (key: string) => Promise<void>;
+}
+
+interface AsyncSourceConfig {
+    storage?: CacheStorage;
+    time?: number;
+    prefix?: string;
```

Comment:
Prefix should be named semantically to provide understating of what this field is stand for 
Suggestion : `cacheKeyPrefix`